src/components/biggive-heading/biggive-heading.tsx

```diff
@@ -1,64 +1,42 @@
 import { Component, Prop, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { headingTag } from '../../globals/heading-tag';
+import { spacingOption } from '../../globals/spacing-option';
 
 @Component({
   tag: 'biggive-heading',
   styleUrl: 'biggive-heading.scss',
   shadow: true,
 })
 export class BiggiveHeading {
-  /**
-   * Space above component
-   */
-  @Prop() spaceAbove: number = 0;
-  /**
-   * Space below component
-   */
-  @Prop() spaceBelow: number = 0;
+  @Prop() spaceAbove: spacingOption = 2;
 
-  /**
-   * Colour Scheme
-   */
-  @Prop() colour: string = 'primary';
+  @Prop() spaceBelow: spacingOption = 4;
 
-  /**
-   * HTML element
-   */
-  @Prop() htmlElement: string = 'h1';
+  @Prop() colour: brandColour = 'primary';
+
+  @Prop() htmlElement: headingTag = 'h1';
 
-  /**
-   * Heading size
-   */
   @Prop() size: number = 1;
```

Comment:
This is another good candidate for a constrained type IMHO, but absolutely no-need to change it as part of this PR.